src/utilities.ts

```diff
@@ -190,6 +190,29 @@ function getWordAtPosition(document, position) {
     return text;
 }
 
+async function loadFileIfNamespaceNotExist(doc) {
+
+    if (getConnectedState()) {
```

Comment:
I'd suggest writing this in a guard-clause fashion, and also to possibly throw, since I would assume that it's an error to try loading a ns that we cannot load. Also, by throwing exceptions, we can inform the caller of why we didn't succeed:

```typescript
async function loadFileIfNamespaceDoesNotExist(doc) {
if (!getConnectedState()) {
   throw new Error("Not connected");
}
cont document = getDocument(doc);
if (!document) {
  throw new Error("Couldn't find " + doc)
}
const client = getSession(getFileType(document));
if (!client) {
  throw new Error("No client available for " + document);
}
const ns = getNamespace(document);
const name = getFileName(document);
const dir = document.hasOwnProperty('fileName') ? path.dirname(document.fileName) : ''; 
const nsList = await client.listNamespaces([]);
if (nsList['ns-list'] && nsList['ns-list'].includes(ns)) {
        return;
}
await client.loadFile(document.getText(), name, dir);
}
```